docs/src/npzd_model_benchmark.md

```diff
@@ -0,0 +1,393 @@
+# [Benchmark: Solution of an NPZD model](@id benchmark-npzd)
+
+We use the NPZD model [`prob_pds_npzd`](@ref) to assess the efficiency of different solvers from [OrdinaryDiffEq.jl](https://docs.sciml.ai/OrdinaryDiffEq/stable/) and [PositiveIntegrators.jl](https://github.com/SKopecz/PositiveIntegrators.jl).
+
+```@example NPZD
+using OrdinaryDiffEq, PositiveIntegrators
```

Comment:
```suggestion
using OrdinaryDiffEqVerner, OrdinaryDiffEqRosenbrock, PositiveIntegrators
```
See above. If I haven't missed anything, we need these two.
Edit: Now I see we need more. But we could import the others further down.